assets/config/0.2.1-coins.json

```diff
@@ -1046,5 +1046,33 @@
     "is_erc_20": false,
     "active": false,
     "currently_enabled": false
+  },
+  "tBTC": {
+    "active": false,
+    "coin": "tBTC",
+    "coingecko_id": "test-coin",
+    "coinpaprika_id": "test-coin",
+    "currently_enabled": false,
+    "electrum": [
+      {
+        "url": "testnet.aranguren.org:51001",
+        "protocol": "TCP"
+      },
+      {
+        "url": "electrum1.cipig.net:10068"
+      },
+      {
+        "url": "electrum2.cipig.net:10068"
+      },
+      {
+        "url": "electrum3.cipig.net:10068"
+      }
+    ],
+    "explorer_url": [
+      "https://blockstream.info/"
```

Comment:
this is wrong @Milerius ! - it is `https://blockstream.info/testnet/` for testnet

CC: @tonymorony